packages/cli/generators/datasource/templates/datasource.ts.ejs

```diff
@@ -20,19 +18,4 @@ export class <%= className %>DataSource extends juggler.DataSource
   ) {
     super(dsConfig);
   }
-
-  /**
-   * Start the datasource when application is started
-   */
-  start(): ValueOrPromise<void> {
-    // Add your logic here to be invoked when the application is started
-  }
-
-  /**
-   * Disconnect the datasource when application is stopped. This allows the
-   * application to be shut down gracefully.
-   */
-  stop(): ValueOrPromise<void> {
-    return super.disconnect();
-  }
```

Comment:
I wonder if we should keep the template with dummy `start` and `stop` to be self-documented. I saw you added:

```
// Observe application's life cycle to disconnect the datasource when
// application is stopped. This allows the application to be shut down
// gracefully. The `stop()` method is inherited from `juggler.DataSource`.
```

Maybe we can add a link to https://loopback.io/doc/en/lb4/Life-cycle.html will do.

Reply:
I prefer to leave out empty start/stop methods, I find them as a distracting boilerplate.

I like the idea of adding a link to our docs, I'll update the comments accordingly.